examples/examples-gpu/nyc-taxi/rf-scikit.ipynb

```diff
@@ -4,47 +4,29 @@
    "cell_type": "markdown",
```

Comment:
<p>"OR need a refresher"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
fixed in https://github.com/saturncloud/examples/pull/58/commits/f092304d317931e1a0f02be86178a58ec58b2bbf